service/common/src/main/java/org/apache/polaris/service/exception/IcebergExceptionMapper.java

```diff
@@ -58,10 +59,24 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.slf4j.event.Level;
+import software.amazon.awssdk.core.exception.SdkException;
 import software.amazon.awssdk.services.s3.model.S3Exception;
 
 @Provider
 public class IcebergExceptionMapper implements ExceptionMapper<RuntimeException> {
+  /** Signifies that we could not extract an HTTP code from a given cloud exception */
+  public static final int UNKNOWN_CLOUD_HTTP_CODE = -1;
+
+  @VisibleForTesting
```

Comment:
Nit: we could remove this tag as it needs to be public for both non-test code and test code.